src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java

```diff
@@ -69,48 +69,55 @@ private static String getVoltageLevelFieldValue(FieldType field, VoltageLevel vo
                 VOLTAGE_LEVEL_ID_2 -> voltageLevel.getId();
             case LOW_VOLTAGE_LIMIT -> String.valueOf(voltageLevel.getLowVoltageLimit());
             case HIGH_VOLTAGE_LIMIT -> String.valueOf(voltageLevel.getHighVoltageLimit());
+            case FREE_PROPERTIES -> voltageLevel.getProperty(propertyName);
```

Comment:
not needed as we add a case in getFieldValue

Reply:
done